src/views/CountryRiskWatch/RiskBarChart/WildfireChart/index.tsx

```diff
@@ -0,0 +1,204 @@
+    return chartPoint;
+}
+
+interface Props {
+    gwisData: RiskData['gwis'] | undefined;
+}
+
+function WildfireChart(props: Props) {
+    const { gwisData } = props;
+
+    const chartContainerRef = useRef<HTMLDivElement>(null);
+    const chartBounds = useSizeTracking(chartContainerRef);
+
+    const points = useMemo(
+        () => {
+            const monthGroupedData = listToGroupList(
+                gwisData?.filter((dataItem) => dataItem.dsr_type === 'monthly') ?? [],
+                (gwisItem) => gwisItem.month,
+            );
+
+            const currentYear = new Date().getFullYear();
+
+            const aggregatedList = mapToList(
+                monthGroupedData,
+                (monthlyData, monthKey) => {
+                    const average = avgSafe(monthlyData.map((dataItem) => dataItem.dsr)) ?? 0;
+                    const min = avgSafe(monthlyData.map((dataItem) => dataItem.dsr_min)) ?? 0;
+                    const max = avgSafe(monthlyData.map((dataItem) => dataItem.dsr_max)) ?? 0;
+
+                    const current = monthlyData.find(
+                        (dataItem) => dataItem.year === currentYear,
+                    )?.dsr;
+
+                    const month = Number(monthKey) - 1;
+
+                    return {
+                        date: new Date(currentYear, month, 1),
+                        month,
+                        min,
+                        max,
+                        average,
+                        current,
+                        maxValue: Math.max(min, max, average, current ?? 0),
+                    };
+                },
+            );
+
+            const maxValue = Math.max(...aggregatedList.map((dataItem) => dataItem.maxValue));
+            const yScale = getScaleFunction(
+                { min: 0, max: maxValue },
+                { min: 0, max: chartBounds.height },
+                { start: chartMargin.top, end: chartMargin.bottom },
+                true,
+            );
+            const xScale = getScaleFunction(
+                { min: 0, max: 12 },
+                { min: 0, max: chartBounds.width },
+                { start: chartMargin.left, end: chartMargin.right },
+            );
+
+            const averagePoints = aggregatedList.map(
+                (dataItem) => ({
+                    x: xScale(dataItem.month),
+                    y: yScale(dataItem.average),
+                }),
+            );
+
+            const minPoints = aggregatedList.map(
+                (dataItem) => ({
+                    x: xScale(dataItem.month),
+                    y: yScale(dataItem.min),
+                }),
+            );
+
+            const maxPoints = aggregatedList.map(
+                (dataItem) => ({
+                    x: xScale(dataItem.month),
+                    y: yScale(dataItem.max),
+                }),
+            );
+
+            const minMaxPoints = [
+                ...minPoints,
+                ...[...maxPoints].reverse(),
```

Comment:
Any reason to copy maxPoints and reverse this array?

Reply:
yes, reverse() mutates the array, an we're creating a closed path